testsuite/.rubocop_todo.yml

```diff
@@ -483,17 +484,17 @@ Style/MissingElse:
     - 'features/step_definitions/retail_steps.rb'
     - 'features/step_definitions/salt_steps.rb'
     - 'features/step_definitions/setup_steps.rb'
+    - 'features/step_definitions/vm_steps.rb'
```

Comment:
I would also disable this other cop for every file.

Reply:
You mentioned this in https://github.com/SUSE/spacewalk/issues/14192#issuecomment-882384462. I did add it to the regular RuboCop configuration.